Tests/SourceKitLSPTests/SwiftCompletionTests.swift

```diff
@@ -864,18 +864,14 @@ final class SwiftCompletionTests: XCTestCase {
           sortText: nil,
           filterText: "myMap(:)",
           insertText: """
-            myMap { ${1:Int} in
-                ${2:Bool}
-            }
+            myMap(${1:\\{ ${2:Int} in ${3:Bool} \\}})
```

Comment:
Similar to my comment in the other test file, i would use raw string literals here so you don’t need to write `\\`.